### ISSUE TITLE
Fix invalid `noqa` comments

### ISSUE THREAD

**Issue by sobolevn** [litestar-org/litestar#1998]
### Description

Ruff (after my fix in https://github.com/astral-sh/ruff/issues/5855) was reporting invalid `noqa` annotations:

```
warning: Invalid `# noqa` directive on /Users/sobolev/Desktop/litestar/tests/unit/test_contrib/test_sqlalchemy/models_bigint.py:19: expected a comma-separated list of codes (e.g., `# noqa: F401, F841`).
warning: Invalid `# noqa` directive on /Users/sobolev/Desktop/litestar/tests/unit/test_contrib/test_sqlalchemy/models_bigint.py:65: expected a comma-separated list of codes (e.g., `# noqa: F401, F841`).
warning: Invalid `# noqa` directive on /Users/sobolev/Desktop/litestar/tests/unit/test_contrib/test_sqlalchemy/models_bigint.py:74: expected a comma-separated list of codes (e.g., `# noqa: F401, F841`).
warning: Invalid `# noqa` directive on /Users/sobolev/Desktop/litestar/tests/unit/test_contrib/test_sqlalchemy/models_uuid.py:22: expected a comma-separated list of codes (e.g., `# noqa: F401, F841`).
warning: Invalid `# noqa` directive on /Users/sobolev/Desktop/litestar/tests/unit/test_contrib/test_sqlalchemy/models_uuid.py:66: expected a comma-separated list of codes (e.g., `# noqa: F401, F841`).
warning: Invalid `# noqa` directive on /Users/sobolev/Desktop/litestar/tests/unit/test_contrib/test_sqlalchemy/models_uuid.py:75: expected a comma-separated list of codes (e.g., `# noqa: F401, F841`).
```

For example, here's the first one: `# pyright: ignore  # noqa: UP`
I think that these `noqa` comments must be fixed.

### Litestar Version

`main`

### Platform

- [ ] Linux
- [X] Mac
- [ ] Windows
- [ ] Other (Please specify in the description above)

## Funding
* If you would like to see an issue prioritized, make a pledge towards it!
* We receive the pledge once the issue is completed & verified

<a href="https://polar.sh/litestar-org/litestar/issues/1998">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/litestar-org/litestar/issues/1998/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/litestar-org/litestar/issues/1998/pledge.svg">
</picture>
</a>

**Comment by Goldziher** [comment]
Yes this is an annoying one for sure. 

@JacobCoffee ?

**Comment by martinlehoux** [comment]
I don't know if I should be posting here, but I have this comment in a file
```
# flake8: noqa: E301
```

and a warning from ruff

```
warning: Invalid code provided to `# ruff: noqa` on line 1: E301
```

**Comment by JacobCoffee** [comment]
Yes. I am thinking that recent changes maybe broke the rule-wide ignores.
```
books: Mapped[List[UUIDBook]] = relationship(  # pyright: ignore  # noqa: UP
```
```
books: Mapped[List[UUIDBook]] = relationship(  # pyright: ignore  # noqa: UP006
```

Also @martinlehoux, I think it is required for the hash to be before noqa as above.. i.e.
```
# flake8 # noqa: E301
```

**Comment by Goldziher** [comment]
fixed in #2044
